src/components/error-page/ErrorPage.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { useStyletron } from "baseui";
+import { styles as s } from "./styles";
+import Block from "./ui/Block";
+import { HeadingXXLarge, ParagraphMedium, ParagraphSmall } from "baseui/typography";
+import { getErrorCodeOverrides, getErrorDescriptionOverrides, getRedirectOverrides } from "./overrides";
+import { ArrowUpIcon } from "../icons";
+import WhiteRect from "./ui/WhiteRect";
+
+export type ErrorPageProps<T extends React.ElementType = React.ElementType> = {
+  errorCode?: number;
```

Comment:
I think we need to provide it. It will be more strictiful in usage but it will be more specific

Reply:
Done.